packages/react/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@web3-onboard/react",
-  "version": "2.1.5",
```

Comment:
@Adamj1232 should we version bump other packages as well?

Reply:
@taylorjdawson which other packages are you thinking?